composer.json

```diff
@@ -22,6 +22,7 @@
     ],
     "require": {
         "php": "^8.2",
+        "bensampo/laravel-enum": "^6.7",
```

Comment:
This is not needed. PHP supports enums out of the box.

https://github.com/BenSampo/laravel-enum#migrate-to-native-php-enums